src/modules/Security.rb

```diff
@@ -363,6 +340,80 @@ def read_kernel_settings
         val = default_value if val.nil? || val == ""
         @Settings[key] = val
       end
+
+      log.debug "Settings (after #{__callee__}): #{@Settings}"
```

Comment:
The original idea was to define log_read_settings but then __method__ or __callee__ is log_read_settings and not the method_name who is calling this method, i have to research more about introspection but i think it is possible.

The string method is more legible, but if you change the method name you don't have to change the log entry, too lazy?